### ISSUE TITLE
Test cases truncated/missing from DG

### ISSUE THREAD

**Issue by kjw142857** [kjw142857/pe#7]
In the DG, under the `Add project` section, the expected result of the `add project` command (with empty input) is truncated. It is possible that other commands below have been omitted as well by the truncation. This may significantly affect the ability of the developer to test the command as they do not know what the result of the empty input `add project` command should be, as well as whether they need to test any other inputs.

The problem also resurfaces in:

- The `Add task` section
- The `Add person to a project` section
- The `Set deadline of a project` section

 

![image.png](https://raw.githubusercontent.com/kjw142857/pe/main/files/f876c220-2d22-4bf8-a86f-05e9964e67f1.png)

`Truncated test cases`

![image.png](https://raw.githubusercontent.com/kjw142857/pe/main/files/bd9285f8-9004-40ba-a723-3041915590d9.png)

![image.png](https://raw.githubusercontent.com/kjw142857/pe/main/files/d865201c-080d-4cf3-8478-d238dd9896c7.png)

![image.png](https://raw.githubusercontent.com/kjw142857/pe/main/files/760cdcc5-c2b7-46af-940d-9f20765ff8a5.png)

**Comment by nus-pe-script** [comment]
# Team's Response
same

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># Cut off documentation in tests in DG
>
>![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2324S2/pe/master/files/9615df56-9a9b-4af6-9192-f640ac9bdac7.png)
>Unable to follow the test case due to this
>
>
>
>
>
><hr>
><sub>[original: nus-cs2103-AY2324S2/pe-interim#4823] [original labels: severity.Medium type.DocumentationBug]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>valid bug. Low severity because it will not affect the normal operations of the app.
# Items for the Tester to Verify
## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)

- [x] I disagree

**Reason for disagreement:** The other tester only covered one instance of the truncation, while I showed it happening 4 separate times throughout the DG with supporting screenshots.

<catcher-end-of-segment><hr>
## :question: Issue severity

Team chose [`severity.Low`]
Originally [`severity.Medium`]

- [x] I disagree

**Reason for disagreement:** If it had happened only once, I would agree that it was `severity.Low` as the developer would only be unable to test a small number of the test cases and the "damage" would have been limited to one command. However, the fact that this happened 4 separate times means a significant number of test cases across various commands were omitted/rendered unreadable for the developer, which is very serious as the developer would now not be able to test a sizable portion of the app correctly. Therefore, `severity.Medium` is accurate.

<catcher-end-of-segment><hr>
